orbit/pkg/constant/constant.go

```diff
@@ -49,4 +49,7 @@ const (
 	UpdateTLSClientKeyFileName = "update_client.key"
 	// SilenceEnrollLogErrorEnvVer is an environment variable name for disabling enroll log errors
 	SilenceEnrollLogErrorEnvVar = "FLEETD_SILENCE_ENROLL_ERROR"
+	// ServerOverridesFileName is the name of the file in the root directory
+	// that specifies the override configuration fetched from the server.
+	ServerOverridesFileName = "server-overrides.json"
```

Comment:
The name `server-overrides` is pretty generic. Is the plan to use this file for other settings in the future?

Reply:
Named it generic in case we want other settings to be controlled by Fleet in the future (currently just the update channels).